assets/css/main.scss

```diff
@@ -0,0 +1,7 @@
+---
+---
+
```

Comment:
I think we should put some comments in there about where the USWDS styles are and were custom styles could go.